packages/core/src/contracts/multisig/basemultisigproxy.ts

```diff
@@ -8,29 +9,40 @@ import { artifacts } from "../generated/artifacts";
 import { createTwoStepSimple, isDecodedLog } from "../utils/contracts";
 import { MultiSigWallet } from "../generated/wrappers/multi_sig_wallet";
 
+/**
+ * Proxies functionionality to a contract instance via multisig wallet. Also supports instantiation *without* multisig proxy, in which case calls are passed directly to the contract.
+ */
 export class BaseMultisigProxy {
   protected ethApi: EthApi;
+  protected contractOwner?: EthAddress;
   protected multisig?: Multisig;
   // TODO(ritave): Add support for lowercase newsroom contract in abi-gen
-  protected instance: OwnableContract;
+  protected instance: Contract | OwnableContract;
 
-  protected constructor(ethApi: EthApi, instance: OwnableContract) {
+  protected constructor(ethApi: EthApi, instance: Contract | OwnableContract, contractOwner?: EthAddress) {
```

Comment:
This contract doesn't need to touch anything concerning Ownable now. `OwnableContract` can be removed altogether

Reply:
This class used to make several calls to `this.instance.owner()` (defined on `OwnableContract` but not `Contract`), but I moved all that logic into the multisigproxy wrapper `create` function, see line `contractOwner = await (instance as OwnableContract).owner.callAsync();` and how `contractOwner` is used here. I think there's a better refactor but as it stands we can't totally do away with `OwnableContract` yet.